### PR TITLE
Drop bluebird dep.

### LINKED ISSUE #7: Drop bluebird for native Promises?
You are using bluebird for `.map` but this should be pretty trivial to write with native promises, would you be open to this? 🍝 💀 

--

cc @paulirish

image

### PR DESCRIPTION
**Pull request by samccone**
Fixes #7

### CODE REVIEW

File: src/frame.js [hunk 1]
```diff
@@ -2,13 +2,21 @@
...
 import fs from 'fs';
 import gmBase from 'gm';
-import Promise from 'bluebird';
 import tempWrite from 'temp-write';
 import getPixelsCb from 'get-pixels';
 import DevtoolsTimelineModel from 'devtools-timeline-model';
 
 const gm = gmBase.subClass({imageMagick: true});
-const getPixels = Promise.promisify(getPixelsCb);
+const getPixels = (buff) => {
```
Comment (sindresorhus) [inline src/frame.js]: You could use [`pify`](https://github.com/sindresorhus/pify) here. It's a tiny package without any dependencies.

Comment (samccone) [inline src/frame.js]: The motivations here is to embrace the native primitives and rely on nothing 💃

**Comment by paulirish** [comment]
needs a rebase now

**Comment by samccone** [comment]
rebased now

**Comment by pmdartus** [comment]
Thanks for the rebase @samccone

**Comment by samccone** [comment]
best
day
ever
:heart:

**Pull request merged**